freetextresponse/tests.py

```diff
@@ -118,103 +118,6 @@ def test_validate_field_data(self, **test_dict):
             validation_list[0].text,
         )
 
-    def test_get_resource_string(self):
```

Comment:
This series of functions was removed because the functionality was folded into build fragment.  Build fragment was removed b/c of an issue with the change to django template as seen in the failing test.    I'll ping you in Slack.